applications/rag/frontend/container/main.py

```diff
@@ -107,19 +103,12 @@ def handlePrompt():
     user_prompt = data['prompt']
     log.info(f"handle user prompt: {user_prompt}")
 
-    context = ""
     try:
-        context = cloud_sql.fetchContext(user_prompt)
-    except Exception as err:
-        error_traceback = traceback.format_exc()
-        log.warn(f"Error: {err}\nTraceback:\n{error_traceback}")
-        warnings.append(f"Error: {err}\nTraceback:\n{error_traceback}")
+        response = {}
+        result = take_chat_turn(llm_chain, session['session_id'], user_prompt)
```

Comment:
Don't we need to write this result back into the chat history table somewhere?

Reply:
Did I also miss where we fetch context now? I don't see a call to similarity_search()

Reply:
@blackzlq - correct. It's done in the CustomVectorStore now.

Reply:
> Don't we need to write this result back into the chat history table somewhere?

This is automatically done by `PostgresChatMessageHistory`